src/plot_api/plot_api.js

```diff
@@ -178,11 +178,15 @@ function plot(gd, data, layout, config) {
         gd.calcdata[i][0].trace = gd._fullData[i];
     }
 
+    function isHidden(gd) {
+        var display = window.getComputedStyle(gd).display;
+        return !display || display === 'none';
+    }
     // make the figure responsive
     if(gd._context.responsive) {
         if(!gd._responsiveChartHandler) {
             // Keep a reference to the resize handler to purge it down the road
-            gd._responsiveChartHandler = function() { Plots.resize(gd); };
+            gd._responsiveChartHandler = function() { if(!isHidden(gd)) Plots.resize(gd).catch(Lib.noop); };
```

Comment:
Do we _really_ need to `.catch` here?

Reply:
No, we don't! The whole point is to not ignore possible rejected promises. Thanks, @etpinard for noticing this.

Fixed in 913bb6877df1e7668748ad4b3e535e38bbf25e6b